setup.cfg

```diff
@@ -12,3 +12,24 @@ versionfile_source = numpy/_version.py
 versionfile_build = numpy/_version.py
 tag_prefix = v
 parentdir_prefix = numpy-
+
+
+[flake8]
```

Comment:
Does it work to add this in `pyproject.toml` instead? `setup.cfg` has to go completely at some point, so best not to add to it.

Reply:
Oh boy, you me and the world wish... https://github.com/PyCQA/flake8/issues/234

The three config locations are setup.cfg, tox.ini or .flake8. tox.ini means no extra file, would you be OK with that?

Reply:
Okay, moved to tox.ini in 134b27aa23b3fd1aee8b8ebbe0fa8bb26ed57847